packages/devtools_app/lib/src/screens/network/network_request_inspector_views.dart

```diff
@@ -2,6 +2,11 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+// ignore_for_file: sort_constructors_first
```

Comment:
i think we can remove this ignore - I'm not seeing anywhere in this file where we are not sorting constructors first, and if there are violations of this, we should just fix those instead.